Projects/App/Sources/Journeys/LoggedInJourney.swift

```diff
@@ -36,7 +36,10 @@ extension AppJourney {
                 case .openCrossSells:
                     CrossSellingScreen.journey { result in
                         if case .openCrossSellingWebUrl(let url) = result {
-                            AppJourney.webRedirect(url: url)
+                            ContinueJourney()
+                                .onPresent {
```

Comment:
Would be nice to move this to a separate function, like we have for webRedirect. since it's reused. Maybe "webRedirectInSafari" or "webRedirectInBroswer"